Pinta.Core/Classes/Layer.cs

```diff
@@ -77,15 +80,20 @@ public Layer (ImageSurface surface, bool hidden, double opacity, string name)
 			set { if (hidden != value) SetValue (HiddenProperty, ref hidden, value); }
 		}
 		
+		public bool Locked {
```

Comment:
Does locking a layer actually do anything currently? If not, I'd suggest filing a bug so that we don't lose track of it before the 1.7 release (adding layer locking functionality would be preferable as a separate pull request, anyways)